utils/swift_build_support/swift_build_support/shell.py

```diff
@@ -88,7 +88,7 @@ def call(command, stderr=None, env=None, dry_run=None, echo=True):
         subprocess.check_call(command, env=_env, stderr=stderr)
     except subprocess.CalledProcessError as e:
         _fatal_error(
-            "command terminated with a non-zero exit status " +
+            "command `{0}` terminated with a non-zero exit status ".format(command) +
```

Comment:
Formatted string?
```suggestion
            f"command `{command}` terminated with a non-zero exit status {str(e.returncode)}, aborting"
```

Then we can avoid concatenating the strings and just format the whole thing. I'll leave it up to you what you want to do. Just looks funny combining the two styles.

Reply:
I don't mind either, just wasn't sure if the Python version that we require supports `f` strings.